ArduPlane/config.h

```diff
@@ -29,7 +29,7 @@
 //
 
 #ifndef ADVANCED_FAILSAFE
- # define ADVANCED_FAILSAFE ENABLED
+ # define ADVANCED_FAILSAFE (!HAL_MINIMIZE_FEATURES && BOARD_FLASH_SIZE > 1024)
```

Comment:
We'd like to remove `HAL_MINIMIZE_FEATURES`.... 

Like parachute, I'm not sure we should start to compile out safety-critical features.